data_steward/cdr_cleaner/clean_cdr_engine.py

```diff
@@ -24,19 +25,7 @@ def add_console_logging(add_handler=True):
     until later.  Useful for debugging.
 
     """
-    logging.basicConfig(
```

Comment:
@ratuagga This is a nice change.  Consider updating the docstring for this function too.  It doesn't seem to be as relevant with this change.

Reply:
Sure @lrwb-aou.